blockchain/vm/interpreter.go

```diff
@@ -51,6 +51,9 @@ type Config struct {
 
 	// Prefetching is true if the EVM is used for prefetching.
 	Prefetching bool
+
+	// Additional EIPs that are to be enabled
+	ExtraEips []int
```

Comment:
I couldn't understand how this values are set. This is only set at `Subtests` which seems to be only used at tests. How are we setting the EIPs?

Reply:
@winnie-byun Yes, you're right. It's usually used in `tests` folder. I couldn't find other usages, but i think this config should be used only in tests cause this *picks* EIP. There's no gaurantee that this is safe.